doc/developer-checklist.md

```diff
@@ -21,7 +21,8 @@ nina.jpg: 6048x4032 uchar, 3 bands, srgb, jpegload
 
 I see:
 
-```bash/usr/bin/time -f %M:%e vips resize nina.jpg x.jpg 0.1
+```bash
```

Comment:
Ooops!

Reply:
It seems that my initial regex that places `bash` for these CLI commands was too greedy. 😅